<launch>
    <group ns="hydrophones">
        <!-- parameters -->
        <node name="param" pkg="hydrophones" type="param.py" respawn="false" output="screen"/>

        <!-- audio -->
        <node name="audio" pkg="hydrophones" type="simulation_audio.py" respawn="false" output="screen"/>

        <!-- tdoa -->
        <node name="tdoa_sim" pkg="hydrophones" type="simulation_tdoa.py" respawn="false" output="screen"/>
        <node name="tdoa" pkg="hydrophones" type="tdoa.py" respawn="false" output="screen"/>

        <!-- solver -->
        <node name="solver" pkg="hydrophones" type="multilateration.py" respawn="false" output="screen"/>

        <!-- error -->
        <node name="error" pkg="hydrophones" type="simulation_error.py" respawn="false" output="screen"/>
    </group>
</launch>
